tools/miqssh/miqhosts-gen

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Query VMDB for MIQ Server hostnames and associated zones and dynamically build the content of miqhosts.
+# Results printed to STDOUT.  Redirect to miqhosts to save.
+
+echo 1>&2
+echo "Redirect standard output of this command to your miqhosts file if desired." 1>&2
+echo "Enter VMDB database password if prompted." 1>&2
+echo 1>&2
+echo "# miqhosts content generated by miqhosts-gen.  See miqhosts-gen to regenerate this file by querying the VMDB if desired."
+echo "#"
+echo "# hostname_or_ip  <white space> groups to assign host to separated by commas."
+echo "#"
+echo "# Lines starting with a # are ignored."
+echo "#"
+
+
+cd /var/www/miq/vmdb
+
+#echo 'select miq_servers.hostname, zones.name from miq_servers join zones on zones.id = miq_servers.zone_id;' | rails db | awk '$1!="hostname" && !/^---/ && !/^\(/ {print}' | sed -e 's/ //g' -e 's/|/ /' -e '/^$/d'
+psql -d vmdb_production -c 'select miq_servers.hostname, zones.name from miq_servers join zones on zones.id = miq_servers.zone_id;' | awk '$1!="hostname" && !/^---/ && !/^\(/ {print}' | sed -e 's/ //g' -e 's/|/ /' -e '/^$/d'
```

Comment:
This could probably be simplified through rails.  What is the output you're looking for?

Reply:
I would like a better way to get this information.  There is a similar psql in miqssh that I don't like.  This one is getting a list of appliances along with the zone each appliance is in.

Reply:
@jeffmcutter If you can supply what you want the output to look like, we can help craft the ruby to get the information for you.